functions/v2/imagemagick/README.md

```diff
@@ -0,0 +1,55 @@
+<img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google Cloud Platform logo" title="Google Cloud Platform" align="right" height="96" width="96"/>
+
+# Google Cloud Functions ImageMagick sample
+
+This sample shows you how to blur an image using ImageMagick in a
+Storage-triggered Cloud Function.
+
+View the [source code][code].
+
+[code]: main.py
+
+## Deploy and Test
+
+1. Follow the [Cloud Functions quickstart guide][quickstart] to setup Cloud
+Functions for your project.
+
+1. Clone this repository:
+
+        git clone https://github.com/GoogleCloudPlatform/python-docs-samples.git
+        cd python-docs-samples/functions/imagemagick
+
+1. Create a Cloud Storage Bucket:
+
+        gsutil mb gs://YOUR_INPUT_BUCKET_NAME
+
+    This storage bucket is used to upload images for the function to check.
+
+1. Create a second Cloud Storage Bucket:
+
+         gsutil mb gs://YOUR_OUTPUT_BUCKET_NAME
+
+     This second storage bucket is used to store blurred images. (Un-blurred images will not be saved to this bucket.)
+
+     This is necessary because saving the blurred image to the input bucket would cause your function to be invoked a second time with the blurred image itself.
+
+1. Deploy the `blur_offensive_images` function with a Storage trigger:
+
+        gcloud functions deploy blur_offensive_images --trigger-bucket=YOUR_INPUT_BUCKET_NAME --set-env-vars BLURRED_BUCKET_NAME=YOUR_OUTPUT_BUCKET_NAME --runtime python39
+        --gen2
```

Comment:
Should `--gen2` be in the same line as the rest of the command?

Reply:
great catch, yes!

Reply:
Done.